### PR TITLE
Made deviceId text instead of integer also fixed the resend attribute

### LINKED ISSUE #1641: Tellstick Binding: Resend parameter does not work & DeviceID not set in Paper UI.
### Usecase goal:
Change the resend count for a device with Paper UI:

### Error Symptom:
Added devices (switch, dimmer) with inbox does not automatically set `deviceID` parameter in PaperUI -> Configure device. After manually changing the `DeviceID` the resend count does not apply.
### Expected behavor:
The deviceID parameter should be auto filled in or deviceID shall not be a required field.
Repeat (resend count) shall reflect what the the tellstick actually send out.

### How to reproduce:
1. Add telldus core bridge
2. Add devices with inbox.
3. Try to configure device in Paper UI.
4. Set DeviceID manually
5. Set Resend count
6. Finishing editing.

## System
2.0.0-SNAPSHOT
Build #683

Tellstick DUO using `telldus-core` bridge.

### PR DESCRIPTION
**Pull request by jarlebh**
This closes #1641

### CODE REVIEW

**Comment by kaikreuzer** [comment]
Thanks! Please remember the [sign-off](https://github.com/openhab/openhab2-addons/blob/master/CONTRIBUTING.md#sign-your-work) in future!

File: addons/binding/org.openhab.binding.tellstick/ESH-INF/thing/devices.xml [hunk 1]
```diff
@@ -72,14 +72,14 @@
...
-			<parameter name="deviceId" type="integer">
+			<parameter name="deviceId" type="text">
 				<label>DeviceID</label>
 				<description>This is the unique id of a device.</description>
 				<required>true</required>
 			</parameter>
 
-			<parameter name="resend" type="integer">
-				<label>Resend</label>
+			<parameter name="repeat" type="integer">
```
Comment (martinvw) [inline addons/binding/org.openhab.binding.tellstick/ESH-INF/thing/devices.xml]: @jarlebh @kaikreuzer is this a backwards compatible change, if not should it be mentioned?

https://github.com/openhab/openhab-distro/wiki/openHAB-2.1-Release-Notes

Comment (kaikreuzer) [inline addons/binding/org.openhab.binding.tellstick/ESH-INF/thing/devices.xml]: Good question, it actually looks incompatible. Also the integer->text change should have an impact on how textual configuration is done. In this context I wonder how all that is reflected in the [full example of the README.md](https://github.com/jarlebh/openhab2-addons/blob/80ebf71419481603bca7ced83f10b1838e7d9ec7/addons/binding/org.openhab.binding.tellstick/README.md#full-example), @jarlebh?

**Comment by jarlebh** [comment]
This does not seem to be documented, should look into that.

Jarle

**Pull request merged**